source/audio/appsVolume.py

```diff
@@ -163,7 +163,15 @@ def _toggleAppsVolumeState():
 	state = _APPS_VOLUME_STATES_ORDER[index]
 	config.conf["audio"]["applicationsVolumeMode"] = state.name
 	_updateAppsVolumeImpl(volume / 100.0, muted, state)
-	ui.message(state.displayString)
+	if state == AppsVolumeAdjusterFlag.ENABLED:
+		# Translators: Reported as a result of the command to allow or disallow to control the volume of other
+		# applications
+		msg = _("Allowed to control the volume of other applications")
+	else:
+		# Translators: Reported as a result of the command to allow or disallow to control the volume of other
+		# applications
+		msg = _("Disallowed to control the volume of other applications")
```

Comment:
Similar to above, but note the use of "is" here. May or may not be best, consistent with other such messages. Can insert "is" in the above suggestion as well if so.

```suggestion
		msg = _("Controlling other applications' volume is disabled")
```

Or, with the action word first. Also a slight difference here: the use of "over" instead of "of". Not sure which is better, though "of" is probably sufficient.

```suggestion
		msg = _("Disabled control over other applications' volume")
```